### DISCUSSION TITLE
Using the mute mic shortcut for another button/event.

### DISCUSSION THREAD

**Discussion by darthbirb** [seerge/g-helper#1015] [Q&A]
Sorry in advance if this isn't the place for this, but I'm not sure who else to ask. I have a 2022 Zephyrus G15

I want to basically set an extra button on my mouse to toggle the microphone mute the same way M3 on the keyboard does. When I record keystrokes, the M1-M4 buttons don't count as actual keypresses for some reason. 

Microsoft Powertoys has a similar feature but it has a presistent toolbar when the mic is muted, which is not ideal. What are my options? If I knew how to, I could also bind the mute mic to say F20 for example, and then binding my mouse button to F20 is trivial.

If anyone has any information please let me know :)

**Comment by seerge** [comment]
[GHelper.zip](https://github.com/seerge/g-helper/files/12269985/GHelper.zip)

This build should do mic-mute from CTRL+SHIFT+F20

**Reply by darthbirb** [reply]
Wow. I can't thank you enough, you're a legend.

I'm assuming I shouldn't install updates? 

Either way, this is exactly what I needed. Works perfectly. Thank you man <3

**Reply by seerge** [reply]
I can keep it in new releases, i don’t think anyone else uses that …

**Reply by darthbirb** [reply]
I would greatly appreciate that!

**Reply by davontragen** [reply]
I was looking for this feature too. Is it possible to bind it to any F-Key. My external keyboard sadly doesn't have a F20 key.

**Reply by darthbirb** [reply]
I've never seen an F20 key personally to be honest. What I did was bind one of my mouse buttons to a Macro of Ctrl+Shift+F20. You can do something similar! If you want a keyboard button to do that, you can use Microsoft PowerToys'  Keyboard Manager Remap feature, and change whatever button you want to act as Ctrl+Shift+F20 instead.

**Reply by davontragen** [reply]
Wow didn't expext that u would answer so fast. Haha. Yeah I think that could work. Thank you very much.
